ethereum/src/ERC20Manager.sol

```diff
@@ -188,14 +188,14 @@ contract ERC20Manager is
      * @param governancePauser_ The address of the GovernanceAdmin contract that will process pauser messages.
      * @param messageQueue_ The address of the message queue contract.
      * @param vftManager The address of the VFT manager contract (on Vara Network).
-     * @param tokens The tokens that will be registered.
+     * @param tokens_ The tokens that will be registered.
      */
     function initialize(
         IGovernance governanceAdmin_,
         IGovernance governancePauser_,
         address messageQueue_,
         bytes32 vftManager,
```

Comment:
```suggestion
        bytes32 vftManager_,
```

Reply:
not need, this is bc of name overlap with view method